docs/source/develop.rst

```diff
@@ -7,31 +7,45 @@ and documentation and style standards are available at the `Dask developer
 guidelines`_ in the main documentation.
 
 .. _Dask: http://dask.org
-.. _`Dask developer guidelines`: http://docs.dask.org/en/latest/develop.html
 
 Install
 -------
 
-After setting up an environment as described in the `Dask developer
-guidelines`_ you can clone this repository with git::
+Clone this repository with git::
 
    git clone git@github.com:dask/distributed.git
+   cd distributed
 
-and install it from source::
+Install all dependencies to match CI:
 
-   cd distributed
-   python setup.py install
+On Linux / MacOSX::
+
+    source continuous_integration/travis/install.sh
+
+On Windows:
 
-Using conda, for example::
+1. Install anaconda or miniconda
+2. ::
+
+    conda create -n distributed python=3.8 tornado=6
+    conda activate distributed
+    conda env update --file continuous_integration/environment-windows.yml
+    python -m pip install .
+
+Alternative instructions, without using conda::
 
    git clone git@github.com:{your-fork}/distributed.git
    cd distributed
-   conda create -y -n distributed python=3.6
-   conda activate distributed
+   python -m venv $HOME/distributed
+   export PATH=$HOME/distributed/bin:$PATH
```

Comment:
Historically we haven't been supporting venv developer workloads.  I'm not sure folks are around to debug issues here if they arise.

Reply:
I'm happy to completely remove the paragraph if you wish. That would mean removing ``dev-requirements.txt`` too.

Reply:
@mrocklin I'd need you to deliberate on this one

Reply:
You don't need me personally.  Any of the maintainers should be able to make this decision.  I think that @martindurant volunteered to be on this week.  I'd also trust @gforsyth  who has been active here.

In general, requiring my activity for something these days in a volunteer capacity is not a winning solution  :)

Reply:
Lets indeed keep with the setup most like the current one, without explicit venv stuff. That could be offered in a later PR, if anyone wants it.

Reply:
@martindurant so should I simply remove the paragraph, but keep dev-requirements.txt there for the time being?

Reply:
Sounds reasonable to me

Reply:
done